frontend/src/components/Header/Header.tsx

```diff
@@ -29,15 +38,29 @@ const THEME_TO_IMAGE = Object.freeze({
 
 interface HeaderProps {
     organisation: OrganisationTheme
+    isAuthenticated: Boolean
 }
 
-function Header({ organisation }: HeaderProps) {
+function Header(props: HeaderProps) {
     const [date, setDate] = useState(new Date().toLocaleString('sv-SE'))
+    const [openLogoutModal, setOpenLogoutModal] = useState(false)
+    const [openHelpModal, setOpenHelpModal] = useState(false)
     const dateIntervalId = useRef<number | undefined>(undefined)
+    const [signOut] = useSignOutMutation();
 
     // No point in showing date and time on mobile devices as they already have a clock in the top right corner
     const showDateAndTime = useMediaQuery<Theme>((theme) => theme.breakpoints.up("lg"));
 
+    const buttonSX = {
+        padding: "0.25rem",
+        mx: "0.5rem",
+        "&:hover": {
+
+            bgcolor: "headerButtons.main",
+            color: "headerButtons.contrastText",
+        },
```

Comment:
I find this hover effect really annoying. Settings just `backgroundColor: rgba(0, 0, 0, 0.4)`, and not modifying `color`, is sufficient.